website/docs/basics/blocks-styles.md

```diff
@@ -49,10 +50,11 @@ export const TypographyEditor = (attributes) => {
 
 use EightshiftLibs\Helpers\Components;
 
+$globalManifest = Components::getManifest(dirname(__DIR__, 2));
 $manifest = Components::getManifest(__DIR__);
 
 $unique = Components::getUnique();
-echo Components::outputCssVariables($attributes, $manifest, $unique); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+echo Components::outputCssVariables($attributes, $manifest, $unique, $globalManifest); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```

Comment:
Should this be escaped or not? Is the output safe?

Reply:
it is safe, I have escaped everything in the function